// lockit/examples/mongodb/config.js

// settings for local MongoDB
exports.db = {
  url: 'mongodb://localhost:27017/',
  name: 'lockit',
  collection: 'users'
};


// exports.emailType = 'nodemailer-smtp-transport';
// exports.emailSettings = {
//   service: 'Mailgun',
//   auth: {
//     user: 'postmaster@username.mailgun.org',
//     pass: 'secret-password'
//   }
// };

exports.signup = {
  route: '/signup',
  tokenExpiration: '1 day',
  views: {
    signup: '',         // input fields 'name', 'email' and 'password' | local variable 'error' | POST /'signup.route'
    linkExpired: '',    // message link has expired | input field 'email' | POST /'signup.route'/resend-verification
    verified: 'login.jade',       // message email is now verified and maybe link to /'login.route'
   // signedUp: 'index.jade',       // message email has been sent => check your inbox
    resend: ''          // input field 'email' | local variable 'error' | POST /'signup.route'/resend-verification
  },
  handleResponse: true  // let lockit handle the response after signup success
};

// settings for MongoDB in cloud (i.e. MongoLab or MongoHQ)
// exports.db = {
//   url: 'mongodb://<user>:<password>@<id>.mongolab.com:<port>/',
//   name: 'lockit',
//   collection: 'users'
// };
